_python/site_parser.py

```diff
@@ -0,0 +1,95 @@
+import psycopg2 as ps
+import bs4
+import pandas as pd
+import requests
+
+class Webscrapper:
+
+    def __init__(self, url: str) -> None:
+        self.url = url
+
+    def __start_connections(self) -> ps.extensions.connection:
+        try:
+            connection = ps.connect(
+                host = "db",
+                port = "5432",
+                dbname = "postgres",
+                password = "1234",
+                user = "postgres"
+            )
+
+            return connection
+        except Exception as e:
+            print(f"Error: {e}")
+
+    def __get_data(self) -> list[tuple[str, str, str]]:
```

Comment:
Usually 2 underscores(__) are put before when you want to hide class attributes. You can check [this](https://realpython.com/python-double-underscore/) article. I think what we want in these methods is 1 underscore(_) to "Indicates that the name is meant for internal use only"